src/rules/converters/no-for-in.ts

```diff
@@ -0,0 +1,14 @@
+import { RuleConverter } from "../converter";
+
+export const convertNoForIn: RuleConverter = tslintRule => {
+    return {
+        rules: [
+            {
+                ...(tslintRule.ruleArguments.length !== 0 && {
+                    ruleArguments: tslintRule.ruleArguments,
+                }),
```

Comment:
Hmm, this isn't quite right. [`no-restricted-syntax`](https://eslint.org/docs/rules/no-restricted-syntax) takes in a list of things it's meant to ban, which are each either a string selector or an object with a string selector and string message. `no-for-in` doesn't provide anything like that.

You'll want to provide a fairly `ruleArguments` that just says to avoid the `for-in` syntax. It looks like your tests start to do that?

Reply:
Oh , so basically we will be good if we fix the `ruleArguments` to `["error", "ForInStatement"]` ?

Reply:
Yup!